### PR TITLE
PaymentsList customer readiness

### PR DESCRIPTION
**Pull request by jakemer**
Steps for Testing/QA
--------------------
Not entirely sure how to test this, since I developed it locally, and am not sure how to match the credentials to the correct env. As well, there's yet no specific way of passing the creds to the component.

Screenshots:
image
image
image
image
image
image

### CODE REVIEW

File: stencil-library/src/api/Api.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export interface IApiResponseCollection<T> extends IApiResponse<T> {
 }
 
 const Api = (authToken: string) => {
-  const apiOrigin = 'http://localhost:3000';
+  const apiOrigin = 'https://justifi.ai';
```
Comment (TiagoFuelber) [inline stencil-library/src/api/Api.ts:26]: This shouldn't be coming from an env file?

File: stencil-library/src/components/payments-list/payments-list.tsx [hunk 2]
```diff
@@ -2,69 +2,125 @@ import { Component, Host, h, Prop, State, Watch } from '@stencil/core';
...
-                <td>{payment.description}</td>
-                <td>{payment.id}</td>
-                <td>{payment.payment_method?.card?.name}</td>
-                <td>{payment.payment_method?.card?.acct_last_four}</td>
-                <td>{payment.status}</td>
-              </tr>;
-            })}
+          <tbody part='table-body'>
+            {
+              this.loading ? this.loadingState :
```
Comment (TiagoFuelber) [inline stencil-library/src/components/payments-list/payments-list.tsx:106]: Nice!

**Review by TiagoFuelber — APPROVED** [review]
Approved with a minor doubt/comment

File: stencil-library/src/components/payments-list/payments-list.tsx [hunk 3]
```diff
@@ -2,69 +2,125 @@ import { Component, Host, h, Prop, State, Watch } from '@stencil/core';
...
+
+    this.loading = false;
+  }
+
+  showEmptyState() {
+    return this.payments ? this.payments.length < 1 : true;
+  }
+
+  emptyState = (
+    <tr part="empty-state">
```
Comment (jakemer) [inline stencil-library/src/components/payments-list/payments-list.tsx:62]: Have you tried using these parts to apply styling? I believe you cannot do ```justifi-payment-form::part(empty-state) td {...}``` for example, meaning you cannot select the child of a part, but only the part directly.

File: stencil-library/src/api/Api.ts (hunk 1 above)
Comment (Cherna) [reply stencil-library/src/api/Api.ts:26]: Good question, I just left it as it was.

File: stencil-library/src/components/payments-list/payments-list.tsx [hunk 4]
```diff
@@ -59,22 +62,22 @@ export class PaymentsList {
...
       </td>
     </tr>
   );
 
   loadingState = (
-    <tr part="loading-state">
-      <td colSpan={8} style={{ textAlign: 'center' }}>
+    <tr>
+      <td part="loading-state" colSpan={8} style={{ textAlign: 'center' }}>
         <div class="spinner-border" role="status">
```
Comment (jakemer) [inline stencil-library/src/components/payments-list/payments-list.tsx:81]: We may want `loading-state-spinner` as well - I can definitely imagine that some folks will want to change the color of the spinner, as well as other things like size.

**Review by TiagoFuelber — APPROVED** [review]

**Pull request merged**